FWCore/ParameterSet/python/Config.py

```diff
@@ -1370,6 +1413,10 @@ def isChosen(self):
     return self.__chosen
   def __and__(self, other):
     return _AndModifier(self,other)
+  def __invert__(self):
+    return _InvertModifier(self)
+  def __or__(self, other):
```

Comment:
These allow `m1 & m2 & m3` to work, but I believe it would fail with `(m1 & m2) & m3` or maybe the failure is `m1 & (m2 & m3)`.  To make the later cases work, I think you need to add `__or__`, `__and__` and `__invert__` to `BoolModifierBase`.

Reply:
The `__or__`, `__and__` and `__invert__` are already part of `_BoolModifierBase`. I added couple of cases like  `(m1 & m2) & m3` to unit tests to actually test that.